src/Integration/RuleSets/SonarRuleSet.cs

```diff
@@ -0,0 +1,38 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.Collections.Generic;
+
+namespace SonarLint.VisualStudio.Integration.RuleSets
+{
+    internal class SonarRuleSet
```

Comment:
I was wondering whether we should use a term other than "ruleset" to avoid confusion with the normal VS usage, and the legacy classes in SonarLint are working with VS rulesets. I don't have any great ideas for new names though - _RuleConfig_, _QualityProfile_?

Reply:
Changed to `QualityProfile`, let's see whether or not we need some prefix (Sonar or SonarLint).